controllers/drplacementcontrol_controller.go

```diff
@@ -470,37 +470,39 @@ func DRPCsFailingOverToCluster(k8sclient client.Client, log logr.Logger, drclust
 
 	drpcCollections := make([]DRPCAndPolicy, 0)
 
-	for drpolicyIdx, drpolicy := range drpolicies.Items {
-		drClusters, err := getDRClusters(context.TODO(), k8sclient, &drpolicies.Items[drpolicyIdx])
-		if err != nil || len(drClusters) <= 1 {
-			log.Error(err, "Failed to get DRClusters")
-
-			return nil, err
-		}
-
-		// Skip if policy is of type metro, fake the from and to cluster
-		if metro, _ := dRPolicySupportsMetro(&drpolicies.Items[drpolicyIdx], drClusters); metro {
-			log.Info("Sync DRPolicy detected, skipping!")
-
-			continue
-		}
+	for drpolicyIdx := range drpolicies.Items {
+		drpolicy := &drpolicies.Items[drpolicyIdx]
 
 		for _, specCluster := range drpolicy.Spec.DRClusters {
```

Comment:
(suggestion) A helper like `func DrpolicyContainsDrcluster(drpolicy *rmn.DRPolicy, clusterName string) bool` can reduce nesting of loops here and improve redability.

Reply:
@ShyamsundarR I added an issue [here](https://github.com/RamenDR/ramen/issues/1220)